storage/flux/table.gen.go.tmpl

```diff
@@ -742,6 +772,10 @@ func (t *{{.name}}GroupTable) advance() bool {
 	aggregate.AccumulateFirst(arr.Timestamps, arr.Values, t.tags)
 	for {
 		arr = t.cur.Next()
+        if t.cur.Err() != nil {
```

Comment:
@davidby-influx formatting for tmpl files needs to be done manually still.

Reply:
> Ah - that is better.
> 
> @davidby-influx does the `influx` command give a non-zero return code? If yes I think that's fine, especially if that error is on stderr instead of stdout.

Returns 1.